ERCS/erc-zk-proof-verification-interface.md

```diff
@@ -0,0 +1,350 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 8039
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.